src/nixpacks/app.rs

```diff
@@ -74,11 +76,13 @@ impl App {
             None => return Ok(Vec::new()),
         };
 
-        let walker = WalkDir::new(&self.source);
+        let walker = WalkBuilder::new(&self.source)
+            .hidden(false)
```

Comment:
This will not copy over hidden files right? I don't think that is the behaviour we want.

Reply:
I think this actually disables ignoring hidden files. It's a stupid way of naming things basically.

Reply:
Well that is confusing af 
![image](https://user-images.githubusercontent.com/3044853/189495537-e7815177-20af-4a0b-93a7-42283f675de3.png)

Want to add a comment so it doesn't accidentally get removed in the future